src/api/entities/BusinessContact.ts

```diff
@@ -0,0 +1,43 @@
+import { Model } from 'objection';
+import { z } from 'zod';
+
+import { BaseModel } from './BaseModel';
+
+export enum ContactType {
+  Business = 'Business',
+  Technical = 'Technical',
+}
+
+export class BusinessContact extends BaseModel {
+  static get tableName() {
+    return 'businessContacts';
+  }
+  static relationMappings = {
+    participant: {
+      relation: Model.BelongsToOneRelation,
+      modelClass: 'Participant',
+      join: {
+        from: 'users.participantId',
+        to: 'participants.id',
+      },
+    },
+  };
+
+  id!: number;
+  name!: string;
+  emailAlias!: string;
+  contactType!: string;
+}
+
+export const BusinessContactSchema = z.object({
+  id: z.number(),
+  name: z.string(),
+  emailAlias: z.string(),
+  contactType: z.nativeEnum(ContactType),
+  participantId: z.number(),
+});
+
+export const BusinessContactsDTO = BusinessContactSchema.omit({
```

Comment:
This looks like it belongs to the router, not the entity - why does the entity want to skip those IDs?